utils/validate_images.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+const core = require('@actions/core');
+const fs = require("fs");
+const path = require('path');
+const isImage = require('is-image');
+const glob = require('glob');
+
+
+const BASE_PATH = './packs/';
+const MAX_SIZE = 4000000;
+const MAX_NUM_IMG = 6;
+const ALLOWED_IMG_EXT = [
+  '.png',
+  '.jpeg',
+  '.jpg',
+  '.svg',
+];
+let valid = true,
+    sizeErrors = [],
+    typeErrors = [],
+    tooManyImages = [];
+    
+/** 
+* Validates an object against a JSON schema
+* @param {String} dir - The directory to parse, set by the BASE_PATH variable
+*/
+
+const globFiles = (dir) => {
+  const entries = [...glob.sync(path.resolve(dir, '**/*'))]
```

Comment:
It's possible to conditionally select files with glob, so you can just pull in the images if you like! You don't have to go with this, just an idea.
```suggestion
  const entries = [...glob.sync(path.resolve(dir, '**/*.+(png|jpeg|jpg|svg)'))]
```

Reply:
The only reason i didn't do this is because of the check to validate they are submitting images that have our allowed extensions, is there a better way to do that?